docs/api.md

```diff
@@ -0,0 +1,34 @@
+# ssh_scan API documentation
+
+**Last Updated:** 2016-09-19 <br>
+**Author:** agaurav77@yahoo.com <br>
+**API Version:** 1
+
+This document explains the ssh_scan Web API, intended to be used in the Web API for ssh_scan.
+
+## Protocol Calls
+
+### scan
+
+**API Call:** `scan` <br>
+**API Method:** `POST`
+
+Scans a target, with an optional port (default 22). Equivalent to `ssh_scan -t host -p port`.
+Parameters:
+* `target` (required), which can be a hostname or an IP address
+
+POST Parameters:
+* `port` on which to scan (optional)
+
+Examples:
+* `/api/v1/scan?host=www.mozilla.org`
+* `/api/v1/scan?host=115.223.33.44`
+  * `port=9999` (POST data)
```

Comment:
This should do a ssh_scan in the backend, so it would show the JSON that ssh_scan returns. For errors, we can check the JSON for the "error" field, and show the corresponding text.